src/fluent/stdout.h

```diff
@@ -46,7 +46,7 @@ class Stdout {
   }
 
  private:
-  std::set<std::tuple<std::string>> deferred_merge_;
+  std::vector<std::tuple<std::string>> deferred_merge_;
```

Comment:
This change was made because the DeferredMerge is calling ra::StreamRaInto, which now accepts a vector...